src/main/java/lithium/university/TwitterConfiguration.java

```diff
@@ -1,8 +1,40 @@
 package lithium.university;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
add Test class for configuration (even though i said beans don't need it, for the labs we're going with close to 100%)